sc2reader/readers.py

```diff
@@ -1723,6 +1724,49 @@ def control_group_update_event(self, data):
             }[data.read_bits(2)](),
         )
 
+class GameEventsReader_38215(GameEventsReader_36442):
+
+    def command_event(self, data):
+        # this function is exactly the same as command_event() from GameEventsReader_36442
+        # with the only change being that flags now has 25 bits instead of 23.
+        return dict(
+            flags=data.read_bits(25),
+            ability=dict(
+                ability_link=data.read_uint16(),
+                ability_command_index=data.read_bits(5),
+                ability_command_data=data.read_uint8() if data.read_bool() else None,
+            ) if data.read_bool() else None,
+            data={  # Choice
+                0: lambda: ('None', None),
+                1: lambda: ('TargetPoint', dict(
+                    point=dict(
+                        x=data.read_bits(20),
+                        y=data.read_bits(20),
+                        z=data.read_uint32() - 2147483648,
```

Comment:
the subtraction is because of https://github.com/Blizzard/s2protocol/blob/master/protocol38215.py#L110

it's a type #84, and in this case the x, y and z are part of a type #93.  https://github.com/Blizzard/s2protocol/blob/master/protocol38215.py#L119